App/components/DatabaseComponent.js

```diff
@@ -69,7 +69,7 @@ export default class DataBaseComponent extends React.Component {
             deleteFromDB();
           }}
         >
-          <Image source={require("./../../assets/del.webp")} style={styles.btnImage} />
+          <MaterialIcons name="delete" size={50} color="white" />
```

Comment:
Adding the icons was a great move. They load faster, as well as make the application look better. Well done!